doc/build/orm/extensions/declarative/relationships.rst

```diff
@@ -155,7 +155,7 @@ the :class:`_schema.MetaData` object used by the declarative base::
     keywords = Table(
         'keywords', Base.metadata,
         Column('author_id', Integer, ForeignKey('authors.id')),
-        Column('keyword_id', Integer, ForeignKey('keywords.id'))
+        Column('keyword_id', Integer, ForeignKey('keyword.id'))
```

Comment:
`keywords` here is an "association table", `Column('keyword_id', Integer, ForeignKey('keywords.id'))` should refer to another table so I replace `keywrods` with `keyword`. Otherwise it will be a "loop"

Reply:
Definitely, it will be better. I fix it in [commit](https://github.com/sqlalchemy/sqlalchemy/pull/5371/commits/7e7d233cf3a0c66980c27db0fcdb3c7d93bc2510)